WcaOnRails/app/assets/stylesheets/competitions.scss

```diff
@@ -105,6 +105,10 @@ $venue-map-wrapper-height: 400px;
   }
 }
 
+#map-wrapper {
+  height: 600px;
```

Comment:
I did, but didn't find good solutions.
One says everything around the map has to be 100% size, but our container is not like that. I don't think it's worth changing the whole thing because of this.